src/main.cpp

```diff
@@ -59,6 +59,14 @@ int main(int argc, char *argv[])
 	// QML-GUI
 	//
 
+	if (qgetenv("QT_QUICK_CONTROLS_STYLE").isEmpty())
+	{
```

Comment:
ah and don't do this; only put the `{` into a new line for functions; Of course this differs from project to project, but I'd like to do it like in the kernel here.